aten/src/ATen/native/cpu/FlashAttentionKernel.cpp

```diff
@@ -21,6 +21,90 @@ namespace at::native {
 
 namespace {
 
+template <typename scalar_t>
+inline void _exp_reduce_sum_fusion_kernel(
```

Comment:
Add a simple math formula describing what this kernel computes?

Reply:
Thanks and added.